resources/debian-package/etc/jitsi/jibri/pjsua.config

```diff
@@ -0,0 +1,31 @@
+--capture-dev=11
+--playback-dev=14
+--video
+--vcapture-dev=1
+--no-color
+--log-level=5
+--app-log-level=5
+--auto-update-nat 0
+--disable-stun
+--no-tcp
+--dis-codec GSM
+--dis-codec H263
+--dis-codec iLBC
+--dis-codec G722
+--dis-codec speex
+--dis-codec pcmu
+--dis-codec pcma
+--dis-codec opus
+--add-codec pcmu
+--add-codec pcma
+--add-codec speex
+--add-codec G722
+--add-codec opus
+--no-vad
+--ec-tail 0
+--quality 10
+#--auto-answer-timer=30
```

Comment:
why are these commened out?

Reply:
If you use sip-jibri from the xmpp side, not sure how it will act ...

Reply:
But then it won't work on the other case... I guess it's ok for now, but maybe we should move those to back the Java side so it can be made functional both ways without changes?

`--max-calls` can stay